resources/zib/zib-MedicationDispense.xml

```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
ext-MedicationDispense.DistributionForm is missing

Reply:
Goed gespot, toegevoegd.